students/etwum/lesson03/list_lab.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+
+# Series 1
+fruit_list = ["Apples", "Pears", "Oranges", "Peaches"]
+
+print(fruit_list)
+
+new_fruit = input("Please input a fruit to add to the list. ")
+
+# adding a new fruit to the end of the fruit_list
+fruit_list.append(new_fruit.capitalize())
+
+print(fruit_list)
+print("----------------------------------------------------------------" + "\n")
+
+# boolean to stop or continue the while loop
+user_number = None
+boolInput_Check = False
+while boolInput_Check == False:
+    # loops until the user inputs a valid number which is less than or equal length of the fruit list
+    try:
+        # get user input in order to display a fruit
+        user_number = int(input("Please input a number to display a fruit: "))
+        if user_number > len(fruit_list):
+            raise Exception
+    # exception raised if the value entered isnt a whole number
+    except ValueError as x:
+        print("Please input a whole number")
+        print(x)
+    # exception raised if number is greater than the list length
+    except Exception:
+        print("The number you entered is too high")
+
+    else:
+        boolInput_Check = True
```

Comment:
really nice touch on input validation!

note that you could also do `while True` and in `else` block put `break`